src/tribol/physics/Mortar.cpp

```diff
@@ -304,59 +305,14 @@ void ComputeSingleMortarGaps( CouplingScheme const * cs )
 
       ++cpID;
 
-      delete [] overlapX;
-
    } // end loop over pairs to compute nodal gaps
 
-   nonmortarMesh.m_nodalFields.m_isGapComputed = true;
```

Comment:
This was removed from `NodalFields` since it wasn't being used and might be confusing since it's not clear if the gaps computed are up-to-date based on the latest geometry.